Jenkinsfile

```diff
@@ -71,20 +76,17 @@ pipeline {
                 }
             }
         }
-        agent {
-          node {
-            label 'ec2-fleet'
-            customWorkspace "docker-images-${BUILD_NUMBER}"
-          }
-        }
         stages {
           stage('Initilize qemu') {
             steps {
               sh """
                 free -h && df -h
                 cat /proc/cpuinfo
                 # initialize qemu
-                docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+                test -z /proc/sys/fs/binfmt_misc/ && \
```

Comment:
racing is possible
[here is how I implemented this step in agent v2 repo](https://github.com/logdna/logdna-agent-v2/blob/8ca55d0aaa2394d686e0b6b1b11b7dcfc301738c/Makefile#L472)